cmd/ui/src/views/Explore/GraphView.tsx

```diff
@@ -194,20 +200,33 @@ const GraphView: FC = () => {
                     id: selectedItem.data.objectid,
                     type: selectedItem.data.nodetype,
                     name: selectedItem.data.name,
+                    graphId: id,
                 })
             );
         }
     };
 
     const options: GraphButtonOptions = { standard: true, sequential: true };
 
+    const nonLayoutButtons: GraphButtonProps[] = [
+        {
+            displayText: 'Search Current Results',
+            onClick: toggleCurrentSearch,
+            disabled: currentSearchOpen
+        }
+    ];
+    
     return (
         <div style={{ position: 'relative', height: '100%', width: '100%', overflow: 'hidden' }} data-testid='explore'>
             <SigmaChart
                 rankDirection={RankDirection.LEFT_RIGHT}
                 options={options}
                 graph={graphologyGraph}
+                currentNodes={currentNodes}
```

Comment:
To me it feels somewhat strange to pass the graphology graph (which details the nodes) and pass `currentNodes` as a separate prop.

Reply:
in order to support both regraph and sigma in the search component, we are passing in the transformed data from the redux store (the output of `transformFlatGraphResponse`), as opposed to data directly from the graph. This is similar to the issue I discussed with Uli above; since we are already running this transformation in the parent component, this prevents us from having to run it in the child as well. 

Definitely open to alternative solutions, I just didn't have a good idea that wasn't messy in other ways!

Reply:
We'll chalk it up as an indicator that this component has some good refactor opportunities